includes/Abilities/Utilities/Utilities.php

```diff
@@ -0,0 +1,404 @@
+<?php
+/**
+ * Utility WordPress Abilities implementation.
+ *
+ * @package WordPress\AI
+ */
+
+declare( strict_types=1 );
+
+namespace WordPress\AI\Abilities\Utilities;
+
+use WP_Error;
+
+/**
+ * Utility WordPress Abilities.
+ *
+ * @since 0.1.0
+ */
+class Utilities {
+
+	/**
+	 * Register any needed hooks.
+	 *
+	 * @since 0.1.0
+	 */
+	public function register(): void {
+		add_action( 'wp_abilities_api_init', array( $this, 'register_abilities' ) );
+	}
+
+	/**
+	 * Registers any needed abilities.
+	 *
+	 * @since 0.1.0
+	 */
+	public function register_abilities(): void {
+		$this->register_get_content_ability();
+		$this->register_get_title_ability();
+		$this->register_get_name_ability();
+		$this->register_get_author_ability();
+		$this->register_get_type_ability();
+		$this->register_get_excerpt_ability();
+		$this->register_get_terms_ability();
+	}
+
+	/**
+	 * Registers the get-content ability.
+	 *
+	 * @since 0.1.0
+	 */
+	private function register_get_content_ability(): void {
+		wp_register_ability(
+			'ai/get-content',
+			array(
+				'label'               => esc_html__( 'Get post content', 'ai' ),
+				'description'         => esc_html__( 'Get the content of a post based on the post ID.', 'ai' ),
+				'category'            => 'ai-experiments',
+				'input_schema'        => array(
+					'type'       => 'object',
+					'properties' => array(
+						'post_id' => array(
+							'type'        => 'integer',
+							'description' => esc_html__( 'The ID of the post to get the content of.', 'ai' ),
+						),
+					),
+					'required'   => array( 'post_id' ),
+				),
+				'output_schema'       => array(
+					'type'        => 'string',
+					'description' => esc_html__( 'The content of the post.', 'ai' ),
+				),
+				'execute_callback'    => static function ( $input ) {
+					$post_id = absint( $input['post_id'] );
+					$post    = self::get_post_object( $post_id );
+
+					if ( is_wp_error( $post ) ) {
+						return $post;
+					}
+
+					return $post->post_content;
+				},
+				'permission_callback' => array( $this, 'permission_callback' ),
+			),
+		);
+	}
+
+	/**
+	 * Registers the get-title ability.
+	 *
+	 * @since 0.1.0
+	 */
+	private function register_get_title_ability(): void {
+		wp_register_ability(
+			'ai/get-title',
```

Comment:
```suggestion
			'ai/get-post-title',
```

Reply:
This ability no longer exists so resolving this